src/main/java/com/onarandombox/MultiversePortals/MultiversePortals.java

```diff
@@ -201,11 +202,11 @@ private void createDefaultPerms() {
     }
 
     public PortalPlayerSession getPortalSession(Player p) {
-        if (this.portalSessions.containsKey(p.getName())) {
-            return this.portalSessions.get(p.getName());
+        if (this.portalSessions.containsKey(p.getUniqueId())) {
+            return this.portalSessions.get(p.getUniqueId());
         }
         PortalPlayerSession session = new PortalPlayerSession(this, p);
-        this.portalSessions.put(p.getName(), session);
+        this.portalSessions.put(p.getUniqueId(), session);
         return session;
```

Comment:
Not very related to this PR, but is it possible that we just create a new PortalPlayerSession for the player on join instead of here. And also remove them when they leave the server? @dumptruckman @nicegamer7 what are your thoughts?

Reply:
We should certainly remove a player's portal session when they leave. But as for creating one when they join, I'm not sure what kind of memory implications that might have...

I think it's a good idea, I'm just worried that large servers might see an uptick in memory usage.

Reply:
Also, if we're removing PlayerPortalSessions when the players leave, there's no need for this change.

We *should* however, change the owner property of portals to be by UUID.